tls-client/README.md

```diff
@@ -36,7 +36,7 @@ Server certificate:
     cert. version     : 3
     serial number     : 65:7B:6D:8D:15:A5:B6:86:87:6B:5E:BC
     issuer name       : C=BE, O=GlobalSign nv-sa, CN=GlobalSign Organization Validation CA - SHA256 - G2
-    subject name      : C=GB, ST=Cambridgeshire, L=Cambridge, O=ARM Ltd, CN=*.mbed.com
```

Comment:
This probably shouldn't change, unless the actual certificate from os.mbed.com has the new branding.